EmailSender.Api/Properties/launchSettings.json

```diff
@@ -8,7 +8,7 @@
         "ASPNETCORE_ENVIRONMENT": "Debug"
       },
       "dotnetRunMessages": true,
-      "applicationUrl": "http://localhost:5002"
+      "applicationUrl": "http://localhost:5005"
```

Comment:
You use the same port in the docker-compose and VS "Debug" mode so developers have to change this port every time. 
I can offer to change this port or to change port in docker-compose